README.md

```diff
@@ -1,83 +1,100 @@
 [![Build Status](https://dev.azure.com/ms/typed-rest-client/_apis/build/status/Microsoft.typed-rest-client?branchName=master)](https://dev.azure.com/ms/typed-rest-client/_build/latest?definitionId=42&branchName=master)
 
-# Typed Rest and HTTTP Client with TypeScript Typings
+# Typed REST and HTTP Client with TypeScript Typings
 
-A lightweight Rest and HTTP Client optimized for use with TypeScript with generics and async await.
+A lightweight REST and HTTP Client optimized for use with TypeScript with generics and async await.
```

Comment:
Should we change this to HTTP client (lowercase) instead of HTTP Client? That seems consistent with everything else